File: Online-Judges/uva/p414.cpp
#include <iostream>
#include<string>
using namespace std;
int cnt[10000];
int main(){
    //freopen("i","r",stdin);
    int n;
    while(true){
        cin >> n;
        if(n==0) {
            return 0;
        }
        for(int i=0;i<n;i++) cnt[0] = 0;

        int maxX = -1;
        for(int i=0;i<n;i++) {
            string s;
            cin >> s;
            for(int j=0;j<25;j++){
                if(s[j]=='X') cnt[i]++;
            }
            maxX = max(maxX,cnt[i]);
        }
        int ans=0;
        for(int i=0;i<n;i++){
            ans = ans + maxX - cnt[i];
        }
        cout<<ans<<endl;
        //printf("%d\n",ans);
    }
    return 0;
}

File: online_judges/uva/p414.py
while True :
    n = int(input())
    if n==0 :
        break
    L = [0 for k in range(n)]
    M = 0;
    for i in range(n):
        s = input()
        for j in range(25):
            if s[j]=='X':
                L[i] += 1
        M = max(M,L[i])
    ans = 0
    for i in range(n):
        ans = ans + M - L[i]
    print(ans)
